sea-query-attr/tests/pass/table_iden.rs

```diff
@@ -0,0 +1,11 @@
+use sea_query::Iden;
+use sea_query_attr::enum_def;
+
+#[enum_def(table_iden = "HelloTable")]
+pub struct Hello {
+    pub name: String,
+}
+
+fn main() {
+    println!("{:?}", HelloIden::Table.to_string());
```

Comment:
Thank you! Can you convert this to `assert_eq`?

Reply:
Sure !